### PR TITLE
(feature) Document how to use GitHub PR review tooling

### PR DESCRIPTION
**Pull request by tahb**
* As the documentation describes this attempts to make it clear how we can conclude wether a decision has consensus or not.
* GitHub settings currently have a minimum of 3 approvals required. Using this as our source of approval rather than :thumbs-up: emojis will offer a single source of truth for consensus. In the same way request for changes over :thumbs-down: will be more helpful as it gives the person a natural place to leave comment beside their opinion.
* Using approvals and request for changes will also serve to document a reversal of decision whereas it will be impossible to track the movement in emojis as a decision is ammended.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,25 @@ dxw staff use this repository as a forum to make and document technical decision
...
+
+### Approve
+
+- This decision SHOULD be accepted
+- A description is optional
+
+### Request changes
+
+- This decision SHOULD NOT be accepted unless the given consequences are acceptable
+- A description is required
```
Comment (erbridge) [inline README.md]: This implies requiring a second look from the reviewer, so they can clear their changes. Do we need to codify that (or a situation in which we are ok to override that and merge anyway, which might require admin rights on this repo).

Comment (tahb) [reply README.md]: I agree that more detail on the part of requesting changes would be helpful, although I'm not clear on what would be a good idea! Generally the concept of a tie-breaker who is probably the CTO or other delegated role may be needed.

If someone requests changes I can see at least the following scenarios:

* a relevant amendment is made 
  - the reviewer preference to approved ✅
  - the reviewer does not agree with the amendment and the cycle restarts with a comment ✅
  - the reviewer does not update their preference perhaps due to being busy or general inactivity - can a tie-breaker dismiss or discount this if it's blocking consensus❓
* no relevant amendment is made
  - the author is busy or inactive - in which case the argument can be made that the proposal has become stale ✅
  - the author does not agree with the feedback or does not know how to proceed - in which case we could be at an impasse if this vote prevented a consensus and require a tie-breaker ❓
  - the team (CTO?) are comfortable with the consequences as outlined ❓

Comment (erbridge) [reply README.md]: Maybe for this round it's sufficient to ask the reviewer to be prepared to respond to comments or changes promptly?

I think we'll need a tie-breaker anyway, but that can separately, I think.

**Comment by Floppy** [comment]
I'm currently installing the tool I have for voting on PRs at https://dxw-developer-rfcs-groupthink.herokuapp.com/...

**Comment by Floppy** [comment]
This proposal is open for discussion and voting. If you are a [contributor](https://dxw-developer-rfcs-groupthink.herokuapp.com/users/) to this repository (and not the proposer), you may vote on whether or not it is accepted.

## How to vote
Vote by entering one of the following symbols in a comment on this pull request. Only your last vote will be counted, and you may change your vote at any time until the change is accepted or closed.

|vote|symbol|type this|points|
|--|--|--|--|
|Yes|:white_check_mark:|`:white_check_mark:`|1|
|No|:negative_squared_cross_mark:|`:negative_squared_cross_mark:`|-1|
|Abstain|:zipper_mouth_face:|`:zipper_mouth_face:`|0|
|Block|:no_entry_sign:|`:no_entry_sign:`|-10|

Proposals will be accepted and merged once they have a total of 3 points when all votes are counted. Votes will be open for a minimum of 7 days, but will be closed if the proposal is not accepted after 90.

Votes are counted [automatically here](https://dxw-developer-rfcs-groupthink.herokuapp.com/proposals/1), and results are set in the merge status checks below.

## Changes

@tahb, if you want to make further changes to this proposal, you can do so by [clicking on the pencil icons here](https://github.com/dxw/developer-rfcs/pull/1/files). If a change is made to the proposal, no votes cast before that change will be counted, and votes must be recast.

**Comment by tahb** [comment]
We discussed in the tech forum (September 6th) that we would start with a voting system that used these GitHub review voting mechanics. We will review a move to another system, potentially Floppy's voting tool - when it's ready.

**Comment by Floppy** [comment]
This proposal is open for discussion and voting. If you are a [contributor](https://dxw-developer-rfcs-groupthink.herokuapp.com/users/) to this repository (and not the proposer), you may vote on whether or not it is accepted.

## How to vote
Vote by entering one of the following symbols in a comment on this pull request. Only your last vote will be counted, and you may change your vote at any time until the change is accepted or closed.

|vote|symbol|type this|points|
|--|--|--|--|
|Yes|:white_check_mark:|`:white_check_mark:`|1|
|No|:negative_squared_cross_mark:|`:negative_squared_cross_mark:`|-1|
|Abstain|:zipper_mouth_face:|`:zipper_mouth_face:`|0|
|Block|:no_entry_sign:|`:no_entry_sign:`|-10|

Proposals will be accepted and merged once they have a total of 3 points when all votes are counted. Votes will be open for a minimum of 7 days, but will be closed if the proposal is not accepted after 90.

Votes are counted [automatically here](https://dxw-developer-rfcs-groupthink.herokuapp.com/proposals/1), and results are set in the merge status checks below.

## Changes

@tahb, if you want to make further changes to this proposal, you can do so by [clicking on the pencil icons here](https://github.com/dxw/developer-rfcs/pull/1/files). If a change is made to the proposal, no votes cast before that change will be counted, and votes must be recast.

**Comment by Floppy** [comment]
This proposal is open for discussion and voting. If you are a [contributor](https://dxw-developer-rfcs-groupthink.herokuapp.com/users/) to this repository (and not the proposer), you may vote on whether or not it is accepted.

## How to vote
Vote by entering one of the following symbols in a comment on this pull request. Only your last vote will be counted, and you may change your vote at any time until the change is accepted or closed.

|vote|symbol|type this|points|
|--|--|--|--|
|Yes|:white_check_mark:|`:white_check_mark:`|1|
|No|:negative_squared_cross_mark:|`:negative_squared_cross_mark:`|-1|
|Abstain|:zipper_mouth_face:|`:zipper_mouth_face:`|0|
|Block|:no_entry_sign:|`:no_entry_sign:`|-10|

Proposals will be accepted and merged once they have a total of 3 points when all votes are counted. Votes will be open for a minimum of 7 days, but will be closed if the proposal is not accepted after 90.

Votes are counted [automatically here](https://dxw-developer-rfcs-groupthink.herokuapp.com/proposals/1), and results are set in the merge status checks below.

## Changes

@tahb, if you want to make further changes to this proposal, you can do so by [clicking on the pencil icons here](https://github.com/dxw/developer-rfcs/pull/1/files). If a change is made to the proposal, no votes cast before that change will be counted, and votes must be recast.

**Comment by Floppy** [comment]
Agreed! (BTW, this vote is also a test of the improvements to Groupthink that I've been working on - managed to give myself a vote at least :) )

Vote: ✅

**Comment by erbridge** [comment]
@Floppy In the readme we stipulate that PRs must be open for at least 14 days after being shared with the team (which isn't necessarily the PR opening date). Can your thing support that?

Personally, I'm against having to learn yet another system for doing reviews. I'd much prefer to keep it in something familiar, which we all use already (GitHub reviews).

**Comment by Floppy** [comment]
Yep, it can. And I totally agree that more familiar tools are good. We talked a bit about splitting out the code that does the build status updates without the special UI, that might be the most useful use of it.

**Pull request merged**